plutus-use-cases/test/Spec/Prism.hs

```diff
@@ -157,6 +158,7 @@ instance ContractModel PrismModel where
     instanceContract _ UserH{} _ = C.subscribeSTO
 
     precondition s (Issue w) = (s ^. contractState . isIssued w) /= Issued  -- Multiple Issue (without Revoke) breaks the contract
+    precondition s (Call _)  = (s ^. contractState . numberOfCalls < 11)
```

Comment:
What happens at 12?

Reply:
Add a comment about this? Would be helpful to know this and why we need txOuts with ada only values.